docs/modules/ROOT/pages/faq.adoc

```diff
@@ -118,6 +118,13 @@ features = [
 ]
 ----
 
+If you are in the early project setup phase and not using anything from the HAL, make sure the HAL is passed to the linker by adding this line to your source:
```

Comment:
```suggestion
If you are in the early project setup phase and not using anything from the HAL, make sure the HAL is explicitly used to prevent the linker removing it as dead code by adding this line to your source:
```

What do you think of this change @jr-oss?

Reply:
Looks good to me.
Thanks for improving the description.